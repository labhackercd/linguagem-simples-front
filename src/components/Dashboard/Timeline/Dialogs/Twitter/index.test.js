import React from 'react';
import { render, screen } from '@testing-library/react';
import TwitterDialog from './index';
import {shallow} from "enzyme/build";

it("should render the Twitter Dialog", () => {
    const component = shallow(<TwitterDialog/>);
    expect(component.exists()).toEqual(true);
    expect(component).toMatchSnapshot();
});
