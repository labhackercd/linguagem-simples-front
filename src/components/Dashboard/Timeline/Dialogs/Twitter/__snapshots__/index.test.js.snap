// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Twitter Dialog 1`] = `
<WithStyles(ForwardRef(Dialog))>
  <WithStyles(ForwardRef(DialogContent))>
    <WithStyles(ForwardRef(DialogContentText))>
      Insira no campo abaixo o link para o tweet desejado
    </WithStyles(ForwardRef(DialogContentText))>
    <WithStyles(ForwardRef(TextField))
      autoFocus={true}
      fullWidth={true}
      id="name"
      label="Link para o tweet"
      margin="dense"
      name="tweetURL"
      onChange={[Function]}
      type="text"
    />
  </WithStyles(ForwardRef(DialogContent))>
  <WithStyles(ForwardRef(DialogActions))>
    <WithStyles(ForwardRef(Button))
      color="primary"
    >
      Cancelar
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="primary"
    >
      Incorporar tweet
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(DialogActions))>
</WithStyles(ForwardRef(Dialog))>
`;
